try:
	dosomething()
except Exception:
	print("Something went wrong")
